yb-voyager/src/utils/utils.go

```diff
@@ -216,6 +216,25 @@ func ParseSchemaAnalysisReport(jsonString string) SchemaReport {
 	return report
 }
 
+func ParseJsonReportFile[T any](path string) (*T, error) {
```

Comment:
Hmm... we already have `utils.JsonFile[T].Load(obj *T) error` for this purpose.

Reply:
oh ok, i was not aware of that.